### PR TITLE
Build instructions bugfix

### LINKED ISSUE #63: Missing step drake_cmake_installed README
The README says:

https://github.com/RobotLocomotion/drake-shambhala/blob/5e8a897c0f53e4f36e777466aa7d9dd38b86b828/drake_cmake_installed/README.md#L47-L49

... but I think we need a `cd drake-shambhala/drake_cmake_installed` between the `git clone` and the `mkdir build` for that to work?  At least, I needed it.

### PR DESCRIPTION
**Pull request by stonier**
Resolves #63 and includes some other minor doc updates.

### CODE REVIEW

File: drake_cmake_installed/src/simple_continuous_time_system/README.md [hunk 1]
```diff
@@ -2,4 +2,17 @@
...
+## How do I run it?
+
+```
+# Switch to the build directory
+cd build/src/simple_continuous_time_system
+# Run the tests
+./simple_continuous_time_system
+```
+
+Note that there is no interaction nor output fromt his program. It is merely intended to exercise the code.
```
Comment (jamiesnape) [inline drake_cmake_installed/src/simple_continuous_time_system/README.md]: `fromt` -> `from`

File: drake_cmake_installed/src/find_resource/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Find Resources
+
+## Description
+
+Simply tests the Drake find resource api from outside of Drake (i.e. finds and loads resources that are part of the Drake install).
```
Comment (jamiesnape) [inline drake_cmake_installed/src/find_resource/README.md]: `api` -> `API`
`i.e.` -> `i.e.,`

File: drake_cmake_installed/src/find_resource/README.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 ## Description
 
-Simply tests the Drake find resource api from outside of Drake (i.e. finds and loads resources that are part of the Drake install).
+Simply tests the Drake find resource API from outside of Drake i.e., finds and loads resources that are part of the Drake install.
```
Comment (jamiesnape) [inline drake_cmake_installed/src/find_resource/README.md]: Missing comma before `i.e.,`

Comment (stonier) [reply drake_cmake_installed/src/find_resource/README.md]: Now, see this is what comes of enforcing grammar rules that aren't even conformed to within the boundary of the United States to British speaking people. 

:all_the_things:

Comment (stonier) [reply drake_cmake_installed/src/find_resource/README.md]: I think I will stop using `i.e.` in protest :)

Comment (jamiesnape) [reply drake_cmake_installed/src/find_resource/README.md]: Comma after i.e. is pretty well enforced in American English and pretty much alien to British English.

Comment (stonier) [reply drake_cmake_installed/src/find_resource/README.md]: I never consumed that much American content, but it was still non-negligible and was surprised to discover this syntax on arriving in Cambridge. There are many web links which note that this is not universally used in the US as well.

I would rather we focus on development than fine-grained US/British differences such as this. Both methods are clear and do not disrupt the flow of the comments to be a consistency issue.

**Pull request merged**